src/porepy/__init__.py

```diff
@@ -196,23 +198,24 @@
     ReferenceVariableValues,
 )
 from porepy.compositional.base import Component, Phase, Fluid
-from porepy.compositional.compositional_mixins import CompositionalVariables, FluidMixin
+from porepy.compositional.compositional_mixins import *
```

Comment:
I don't feel I know the impact/usage of this, so I'm not sure my opinion is very well founded. Would an option to include only a subset to top-level and in files using compositional extensively simply import compositional explicitly?

Reply:
I guess the reason for keeping the variables and fluidmixin is that these will show up in all models, since all models in a sense are compositional? If correct, please add a short comment to this effect, so that I don't get confused about what is so special about these classes whenever I look at this line.

Reply:
All models have the fluid mixin, that is correct. But only truly multiphase multicomponent models require the `CompositionalVariables`. Do you want only the `FluidMixin` included on the top level?